sdk/package.json

```diff
@@ -45,6 +45,7 @@
     "jwt-decode": "^3.1.2",
     "lru-memorise": "0.3.0",
     "magic-sdk": "^21.2.0",
+    "merkletreejs": "^0.3.11",
```

Comment:
Was wondering if the dependency needs to be added to the SDK package.json as well, if it's already listed in the orderbook package.json dependencies. Would the orderbook not already provide the dependency code when building, similar to other dependencies for other packages?

Reply:
This is a consequence of `yarn build`, I did not add it manually - any dependency in that package.json's of the packages gets lifted here

Reply:
I see, thank you for the clarification. Apologies, was not aware of this 😄